INSTALLATION.rst

```diff
@@ -61,59 +63,89 @@ Recommended Method
 
    In addition to the default packages, many other developers provide
    "channels" to distribute their own builds of a variety of software.
-   These days, the most useful channel is `conda-forge`, which provides
+   These days, the most useful channel is conda-forge_, which provides
    everything necessary to install :term:`FiPy`.
 
 * `install Miniconda`_ on your computer
-* run::
+* Install :term:`FiPy` :ref:`REQUIREDPACKAGES` with one of the following:
 
-    $ conda create --name <MYFIPYENV> --channel conda-forge python=<PYTHONVERSION> fipy gmsh
+    * Conda environment files::
```

Comment:
This option will pin all the packages so is the most reproducible, but not particularly upgradable. For most, this is the safest way to generate a FiPy environment that actually works.

Reply:
Reverse that. Opposite way around.

Reply:
This option is the most upgradable in the future and probably the best for development.